src/components/Icon/Icon.story.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Icon from './Icon';
+import paths from './Icons';
+
+const stories = storiesOf('Icons', module);
+
+stories.add('Icons', () => (
+  <ul className="w-full max-w-full">
+    {Object.keys(paths).map((icon) => (
+      <li className="inline-block text-center w-1/4">
+        <Icon className="w-full" key={icon} kind={icon} />
```

Comment:
use the `width` and `heigth` props here and add a knob (default to 100%) so users can see the how the adjustments affect the icons. Using the `w-full` here is a little confusing because it overrides the style of the svg so the width scales but not the height